.taskcat.yml

```
@@ -17,6 +17,7 @@ tests:
       QSS3BucketRegion: $[taskcat_current_region]
       ClusterName: tcat-$[taskcat_random-string]-$[taskcat_current_region]
       ClusterRegion: US_EAST_1
+      DatabasePassword: override
```

Can we change this to something like below? Do you have any restrictions on the password values?
```suggestion
      DatabasePassword: $[taskcat_genpass_12]
```